libraries/app/database_api.cpp

```diff
@@ -2071,8 +2071,9 @@ vector<withdraw_permission_object> database_api_impl::get_withdrawals_giver(acco
    FC_ASSERT( limit <= 100 );
    vector<withdraw_permission_object> result;
 
+   auto withdraw_index_end = _db.get_index_type<withdraw_permission_index>().indices().get<by_from>().end();
```

Comment:
Two lines of `_db.get_index_type<withdraw_permission_index>().indices().get<by_from>()` is ugly.

Reply:
was just needing `const auto&` to make it work.